docs/06-appearance.md.lua

```diff
@@ -229,8 +70,9 @@ f:write[[
 
 Beautiful is where Awesome theme variables are stored.
 
+<br \><br \>
```

Comment:
Shouldn't it be `<br />` instead of `<br \>`?

Reply:
done